test/graphql/attachments_query_test.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class AttachmentsQueryTest < ActiveSupport::TestCase
+  ATTACHMENTS_QUERY = <<~GRAPHQL
+    query($first: Int) {
+      samples(first: $first) {
+        nodes {
+          id
+          attachments {
+            edges {
+              node {
+                id,
+                metadata,
+                filename,
+                byteSize
+              }
+            }
+          }
+        }
+      }
+    }
+  GRAPHQL
+
+  ATTACHMENTS_URL_QUERY = <<~GRAPHQL
+    query($first: Int) {
+      samples(first: $first) {
+        nodes {
+          id
+          attachments {
+            edges {
+              node {
+                id,
+                filename,
+                attachmentUrl
+              }
+            }
+          }
+        }
+      }
+    }
+  GRAPHQL
+
+  def setup
+    @user = users(:john_doe)
+    @sample = samples(:sample1)
+  end
+
```

Comment:
Can you also add a test for getting the attachment metadata with the optional array of keys to limit which metadata is returned?

Reply:
Added in https://github.com/phac-nml/irida-next/pull/408/commits/a9e7880fdfd0d774e15793238984a29ab352d2cd